### PR TITLE
Registering yashmantri20

### PR DESCRIPTION
**Pull request by yashmantri20**
Hey Maintainers,

Kindly accept my request 

Thank You

### CODE REVIEW

File: content/participants/ttjc@yashmantri20/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+_description_ The tracker gives the information regarding total positive cases , total active cases , total recovered , total deaths , it also includes a self assessment test to check whether you are safe or not and gives information regarding symptoms , testing and state wise data .
+
+_stack_ ReactJs , NodeJs , HTML and CSS.
+
+_hosted Link_ https://covid19-india-app.netlify.com
+
+_github link_ https://github.com/yashmantri20/covid-19
+
+#### Musicee : Motion Detection
```
Comment (khusharth) [inline content/participants/ttjc@yashmantri20/index.md:66]: Hey! @yashmantri20 this project was done before the start of jobChallenge. Only projects made during the challenge are accepted. Please remove it.

Comment (yashmantri20) [reply content/participants/ttjc@yashmantri20/index.md:66]: I have to remove motion detection project right ??

Comment (yashmantri20) [reply content/participants/ttjc@yashmantri20/index.md]: I have removed that project

**Comment by khusharth** [comment]
@gkanishk Good to merge.

**Pull request merged**